### PR TITLE
Add ZIP 320 test vectors.

### PR DESCRIPTION
**Pull request by nuttycom**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -43,6 +43,7 @@ zip_0244 = "zcash_test_vectors.zip_0244:main"
 # Transparent test vectors
 bip_0032 = "zcash_test_vectors.transparent.bip_0032:main"
 zip_0316 = "zcash_test_vectors.transparent.zip_0316:main"
+zip_0320 = "zcash_test_vectors.zip_0320:main"
```
Comment (daira) [inline pyproject.toml:46]: Should this be in the transparent module?

File: test-vectors/json/zip_0320.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+[
+    ["From https://github.com/zcash-hackworks/zcash-test-vectors/blob/master/unified_address.py"],
```
Comment (daira) [inline test-vectors/json/zip_0320.json:2]: This comment string is wrong.

File: zcash_test_vectors/transparent/zip_0320.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    converted = convertbits(p2pkh_bytes, 8, 5)
+    return bech32_encode("tex", converted, Encoding.BECH32M)
+
+def decode(hrp_expected, tex_addr):
+    (hrp, data, encoding) = bech32_decode(tex_addr)
+    if data is None or encoding != Encoding.BECH32M:
+        raise InvalidEncoding("ZIP 320 addresses must be encoded using Bech32m")
+    if hrp != hrp_expected:
+        raise HrpMismatch("Expected: " + hrp_expected + "; got " + hrp)
+    return bytes(convertbits(data, 6, 8, False))
```
Comment (daira) [inline zcash_test_vectors/transparent/zip_0320.py:18]:
```suggestion
    return bytes(convertbits(data, 5, 8, False))
```

File: zcash_test_vectors/zip_0320.py [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+                't_addr': t_addr,
+                'p2pkh_bytes': p2pkh_bytes,
+                'tex_addr': tex_addr,
+                'account': account,
+                'child_index': j,
+            })
+
+    render_tv(
+        args,
+        'unified_address',
```
Comment (daira) [inline zcash_test_vectors/zip_0320.py:52]: This is wrong.

**Review by daira — CHANGES_REQUESTED** [review]
Changes needed.

**Comment by nuttycom** [comment]
[force-pushed](https://github.com/zcash/zcash-test-vectors/compare/6167963..439d5a3) to address comments from code review.

File: test-vectors/json/zip_0320.json [hunk 5]
```diff
@@ -0,0 +1,19 @@
+[
+    ["From https://github.com/zcash-hackworks/zcash-test-vectors/blob/master/transparent/zip_0320.py"],
```
Comment (daira) [inline test-vectors/json/zip_0320.json:2]: This URL is not correct, it should be https://github.com/zcash/zcash-test-vectors/blob/master/zcash_test_vectors/transparent/zip_0320.py

**Review by daira — APPROVED** [review]
utACK modulo the still-incorrect URLs.

**Review by daira — APPROVED** [review]
utACK

**Pull request merged**